packages/app-mobile/components/NoteBodyViewer/hooks/useSource.ts

```diff
@@ -203,7 +212,7 @@ export default function useSource(noteBody: string, noteMarkupLanguage: number,
 						<meta charset="UTF-8">
 						<meta name="viewport" content="width=device-width, initial-scale=1">
 						<style>
-							${shim.mobilePlatform() === 'ios' ? iOSSpecificCss : ''}
+							${shim.mobilePlatform() === 'ios' ? iOSSpecificCss : defaultCss}
```

Comment:
Here make it `${shim.mobilePlatform() === 'ios' ? iOSSpecificCss + '\n' + defaultCss: defaultCss}`